tests/test_detectionlists_actions.py

```diff
@@ -340,125 +294,86 @@ def test_handle_action_when_add_high_risk_employee_adds_response_to_data(
         )
-        result = ActionResult(dict(param))
-        add_data_mock = mocker.MagicMock()
-        result.add_data = add_data_mock
-        mock_result_adder.return_value = result
+
         connector = _create_add_hr_connector(mock_py42_with_user)
         connector.handle_action(param)
-        add_data_mock.assert_called_once_with(response_data)
+        assert_successful_single_data(connector, response_data)
 
     def test_handle_action_when_add_high_risk_employee_and_is_successful_sets_success_status(
-        self, mocker, mock_py42_with_user, mock_result_adder
+        self, mock_py42_with_user
     ):
         param = {"username": "test@example.com"}
-        result = ActionResult(dict(param))
-        set_status_mock = mocker.MagicMock()
-        result.set_status = set_status_mock
-        mock_result_adder.return_value = result
         connector = _create_add_hr_connector(mock_py42_with_user)
         connector.handle_action(param)
-        expected_message = "test@example.com was added to the high risk employees list"
-        set_status_mock.assert_called_once_with(1, expected_message)
+        assert_successful_message(connector, "test@example.com was added to the high risk employees list")
 
     def test_handle_action_when_remove_high_risk_employee_calls_remove_with_expected_args(
-        self, mock_py42_with_user, mock_result_adder
+        self, mock_py42_with_user
     ):
         param = {"username": "test@example.com"}
-        result = ActionResult(dict(param))
-        mock_result_adder.return_value = result
         connector = _create_remove_hr_connector(mock_py42_with_user)
         connector.handle_action(param)
         mock_py42_with_user.detectionlists.high_risk_employee.remove.assert_called_once_with(
             "TEST_USER_UID"
         )
+        assert_success(connector)
 
     def test_handle_action_when_remove_high_risk_employee_adds_response_to_data(
-        self, mocker, mock_py42_with_user, mock_result_adder
+        self, mocker, mock_py42_with_user
     ):
         param = {"username": "test@example.com"}
 
         # This API call does not have response data.
         mock_py42_with_user.detectionlists.high_risk_employee.remove.return_value = create_mock_response(
             mocker, {}
         )
-        result = ActionResult(dict(param))
-        add_data_mock = mocker.MagicMock()
-        result.add_data = add_data_mock
-        mock_result_adder.return_value = result
         connector = _create_remove_hr_connector(mock_py42_with_user)
         connector.handle_action(param)
-        add_data_mock.assert_called_once_with({"userId": _TEST_USER_UID})
+        assert_successful_single_data(connector, {"userId": _TEST_USER_UID})
 
-    def test_handle_action_when_remove_high_risk_employee_and_is_successful_sets_success_status(
-        self, mocker, mock_py42_with_user, mock_result_adder
+    def test_handle_action_when_remove_departing_employee_and_is_successful_sets_success_status(
```

Comment:
This test is apparently a duplicate test? It is also defined on line 220

Reply:
Probably should be `high_risk_employee` instead of `departing` in the test sig.

Reply:
it has the incorrect name, should be hr instead of de